x-pack/plugins/viz_editor/server/query/sanitize_query.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Query, SelectOperation } from '../../common';
+import { selectOperations } from './select_operations';
+
+// If size is not specified in the query, we'll use this
+const DEFAULT_SIZE = 100;
```

Comment:
This differs from the type docs where you set the default to 1000. Maybe we need a constant in the type file?

Reply:
Good call.